ml/src/main/scala/ai/h2o/sparkling/ml/algos/regression/DistributionForRegressionCheck.scala

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Maybe I looked at wrong places, I couldn't find any usage of this class

Reply:
It's hidden in code generation ;-) See https://github.com/h2oai/sparkling-water/pull/2211/files#diff-af3d14b640ff91fe5eb78f29d5400005R127 and https://github.com/h2oai/sparkling-water/pull/2211/files#diff-881337a409b96c7e9e252b23255d55a1R39

Reply:
haha, thanks!